google-cloud-datastore/acceptance/datastore/datastore_test.rb

```diff
@@ -388,12 +388,12 @@
 
     let(:jonsnow) do
       character = Google::Cloud::Datastore::Entity.new.tap do |e|
-        e["name"]        = "Jon Snow"
-        e["family"]      = "Stark"
+        e["name"]        = "Jon"
+        e["family"]      = "Snow"
```

Comment:
Lol. I needed to add another character from a different family for a test case, and it was bothering me that the dataset had just first names for all characters ("Arya", "Bran" etc), but full name for Jon ("Jon Show"). So I moved his last name to the `family` field.

I just updated the code to show Jon as from "Targaryen" family. Since this code was originally written 8 years ago, it seemed correct to update it with the info from the latter seasons.

[I hope GRRM reads this discussion thread & gets motivated to finish the remaining 2 books 😥]